src/constants/MarkdownFiles/posts/2025-07-14-gsoc-25-BishoyWadea-week06.md

```diff
@@ -0,0 +1,119 @@
+---
+title: "GSoC ’25 Week 06 Update by Bishoy Wadea"
+excerpt: "Mid term evaluation reflection"
+category: "DEVELOPER NEWS"
+date: "2025-07-13"
+slug: "gsoc-25-BishoyWadea-week06"
+author: "@/constants/MarkdownFiles/authors/bishoy-wadea.md"
+tags: "gsoc25,sugarlabs,week06,midterm,BishoyWadea"
+image: "assets/Images/GSOC.png"
+---
+
+<!-- markdownlint-disable -->
+
+# Week 06 Progress Report by Bishoy Wadea
+
+**Project:** [Euclid’s Game](https://github.com/Bishoywadea/Euclid-s-Game)  
+**Mentors:** [Ibiam Chihurumnaya](https://github.com/chimosky)  
+**Assisting Mentors:** [Walter Bender](https://github.com/walterbender/)  
+**Reporting Period:** 2025-06-07 - 2025-07-14 
+
+---
+
+## Goals for This Week
+
+- **Goal 1:** Start implementing Euclid’s Game
+---
+
+## This Week’s Achievements
+
+### *Goal 1: add helpful video tutorial in Soma Cube Game*
+
+1. **add video in help button**  
+   - commit: [modify code to handle help videos](https://github.com/Bishoywadea/Soma-Cube/commit/63a7daaa8009f5f54791cdf9081e765846135f70)
+
+[![illustration tutorial video](https://i9.ytimg.com/vi_webp/Q4BKp3Yo3Uw/mq3.webp?sqp=CPDv2sMG-oaymwEmCMACELQB8quKqQMa8AEB-AH-CYAC0AWKAgwIABABGGUgTihBMA8=&rs=AOn4CLCEh8DLiBCY1blowOssb9l5JiRdRQ)](https://youtu.be/Q4BKp3Yo3Uw)
```

Comment:
There is a way to embed a YT video with the markdown.

Please see the template: https://github.com/sugarlabs/www-v2/blob/main/src/constants/MarkdownFiles/posts/YYYY-MM-DD-GSoC_DMP_SSoC_Template.md

Example:

```
Brief summary or a video. [youtube: MM-H69cHYMk]
```